http-server/src/main/java/io/micronaut/http/server/util/HttpLocaleResolver.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.util;
+
+import io.micronaut.core.util.LocaleResolver;
+import io.micronaut.http.HttpRequest;
+
+/**
+ * Responsible for determining the current locale from a request.
+ *
+ * @author James Kleeh
+ * @since 2.3.0
+ */
+public interface HttpLocaleResolver extends LocaleResolver<HttpRequest<?>> {
```

Comment:
Should be annotated with `@Indexed(HttpLocalResolver.class)` since there can be multiple